### PR TITLE
Support language code

### PR DESCRIPTION
**Pull request by Chentong-Xiang**

### CODE REVIEW

**Review by njerigrevious — APPROVED** [review]
This looks good - thank you!

File: concepts/search-concept-acceptlanguage-header.md [hunk 1]
```diff
@@ -0,0 +1,144 @@
...
+|zh-Hant|Chinese (Traditional)|
+|zu|isiZulu|
+
+## Example
+The following example shows the Accept-Language header with the value `en-us`. Only the first value provided is used for language-related processing.
+
+```http
+POST /search/query
+Content-Type: application/json
+Accept-Language: en-us
```
Comment (Lauragra) [inline concepts/search-concept-acceptlanguage-header.md:136]: In the reference content, do you list this as an optional header in the Request headers H2 section? I spot-checked a few topics and I don't see it listed/used, and I also see a few topics under Search that are missing the required Request headers H2 section. As part of this PR, please update the reference content accordingly to list and describe this header as part of the API request, and to link to this topic for the list if values.

File: concepts/search-concept-acceptlanguage-header.md [hunk 2]
```diff
@@ -0,0 +1,144 @@
...
+|vi|Vietnamese|
+|wo|Wolof|
+|xh|isiXhosa|
+|yo-Latn|Yoruba|
+|zh-Hans|Chinese (Simplified)|
+|zh-Hant|Chinese (Traditional)|
+|zu|isiZulu|
+
+## Example
+The following example shows the Accept-Language header with the value `en-us`. Only the first value provided is used for language-related processing.
```
Comment (Lauragra) [inline concepts/search-concept-acceptlanguage-header.md:131]: Does the header accept multiple values or is a single value required? Please clarify. Also, please be sure that your examples in the reference content include this header, as applicable.

File: concepts/search-concept-acceptlanguage-header.md [hunk 3]
```diff
@@ -0,0 +1,144 @@
...
+
+```http
+POST /search/query
+Content-Type: application/json
+Accept-Language: en-us
+```
+
+## Known limitations
+- Compliance with HTTP standards.
```
Comment (Lauragra) [inline concepts/search-concept-acceptlanguage-header.md:140]: It's not clear how this is a limitation. Is your intent to convey that the header is not in compliance with HTTP standards, and how so? Please elaborate.

File: concepts/search-concept-acceptlanguage-header.md [hunk 4]
```diff
@@ -0,0 +1,144 @@
+---
+title: "Supported language codes"
```
Comment (Lauragra) [inline concepts/search-concept-acceptlanguage-header.md:2]: Please surface this topic in the TOC by adding an entry to the toc.yml file in the Concepts folder. I assume you want this topic to be under Search in the Learn section of the TOC, with the label "Supported language codes".

File: concepts/search-concept-acceptlanguage-header.md (hunk 1 above)
Comment (Chentong-Xiang) [reply concepts/search-concept-acceptlanguage-header.md]: It's an optional header in request header, so we just want to hand-up to say we support this language setting.

File: concepts/search-concept-acceptlanguage-header.md (hunk 2 above)
Comment (Chentong-Xiang) [reply concepts/search-concept-acceptlanguage-header.md]: Thanks for pointing out. It can be multiple values and update this.

**Review by Lauragra — APPROVED** [review]

**Pull request merged**